src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/CommandBar.xaml

```diff
@@ -11,7 +11,7 @@
     xmlns:help="using:Microsoft.CmdPal.UI.Helpers"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:ui="using:CommunityToolkit.WinUI"
-    xmlns:viewmodels="using:Microsoft.CmdPal.UI.ViewModels"
+    xmlns:viewModels="using:Microsoft.CmdPal.UI.ViewModels"
```

Comment:
There are no particular rules about `xmlns:...`, but as a general rule, it's helpful to be consistent across a project. I'm picking camelCase because it happens to make other things happy (but UpperCamel would also work and was also sometimes used...).